website/content/docs/agent/config/cli-flags.mdx

```diff
@@ -482,6 +482,8 @@ information.
 
 - `-node` ((#\_node)) - The name of this node in the cluster. This must
   be unique within the cluster. By default this is the hostname of the machine.
+  This cannot contain whitespace or quotes, and in order for the node to be
+  discoverable via DNS it must only contain alphanumerics and dashes (`-`).
```

Comment:
```suggestion
  The node name cannot contain whitespace or quotation marks. To query the node from DNS, the name must only contain alphanumeric characters and hyphens (`-`).
```
I know these are kind of nitpick-y, but _quotes_ are the words inside of the _quotation marks_. We tend to misuse and overuse _via_, which means going from point A to point C by passing through point B. Finally, a single `-` is a _hyphen_. It's improbable that someone would misinterpret _dash_ to mean em dash, underscore, back-tick, etc., but it's probably best to use the correct word.